watchman/decorators.py

```diff
@@ -66,14 +66,19 @@ def _get_passed_token(request):
         return token
 
     def _validate_token(request):
-        watchman_token = settings.WATCHMAN_TOKEN
-
-        if watchman_token is None:
+        if settings.WATCHMAN_TOKENS:
+            watchman_tokens = settings.WATCHMAN_TOKENS.split(',')
+        elif settings.WATCHMAN_TOKEN:
+            watchman_tokens = [settings.WATCHMAN_TOKEN, ]
+        else:
+            watchman_tokens = []
+
+        if not watchman_tokens:
```

Comment:
why not set `watchman_tokens = None` in line 74 and check `if watchman_tokens is None`?